src/garvis/heartbeat_cli.py

```
@@ -27,15 +27,17 @@ def build_parser() -> argparse.ArgumentParser:
     parser.add_argument("--root", type=Path, default=None)
     sub = parser.add_subparsers(dest="command", required=True)
     sub.add_parser("run-once")
-    sub.add_parser("daemon")
+    daemon = sub.add_parser("daemon")
+    daemon.add_argument("--speak", action="store_true")
     sub.add_parser("health")
     return parser
 
 
 def main(argv: Sequence[str] = None) -> int:
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)